fuzzers/afltriple/fuzzer.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLFast fuzzer."""
```

Comment:
s/AFLFast/afltriple

Reply:
I reused code from AFLFast and the code is removed.

Reply:
The changes is spited into three specific commits and summarized.

Reply:
Sorry I'm not sure what this means. But I think Abhishek is asking what new techniques/changes does afltriple have that make it worth testing. As far as I can tell, afltriple has two commits. One seems to be afl2.52b the other is https://github.com/datadancer/afltriple/commit/96cdb1d078986a749f785214262cd6b300851a1f which is a relatively small instrumentation change. Is there more that was changed or just this? What is this commit doing differently from default AFL?